common/v2/features/SendAssets/components/TransactionReceipt.tsx

```diff
@@ -37,15 +38,19 @@ export default function TransactionReceipt({ txReceipt, txConfig }: IStepCompone
     );
```

Comment:
Since we already `return` if `txReceipt` is undefined, is this ternary really necessary?